jekyll/_cci2/version-control-system-integration-overview.adoc

```diff
@@ -54,53 +55,55 @@ GitHub can be integrated into your CircleCI org in one of two ways, via the GitH
 | Feature | GitHub App | GitLab | GitLab Self-Managed | GitHub OAuth | Bitbucket Cloud | Bitbucket Data Center | CircleCI server
 
 | link:https://circleci.com/changelog/changes-to-code-checkout-for-orgs-that-integrate-with-github-app/[Secure HTTPS checkout]
-| icon:thumbs-up[role="circle-green"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
+| [.circle-green]#**Yes**#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
+| [.circle-green]#**Yes** ^2^#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
 
 | Reduced permissions when integrating with CircleCI. Select a subset of repositories to be used with CircleCI.
-| icon:thumbs-up[role="circle-green"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
-| icon:thumbs-down[role="circle-red"]
+| [.circle-green]#**Yes**#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
+| [.circle-green]#**Yes** ^2^#
+| [.circle-red]#**No**#
+| [.circle-red]#**No**#
```

Comment:
I think we can consider this a YES, since it works through individual project tokens

Reply:
(BBDC)